duckdb_engine/__init__.py

```diff
@@ -185,5 +185,11 @@ def do_begin(self, connection: ConnectionWrapper) -> None:
     def get_dialect_cls(cls, u: str) -> Type["Dialect"]:
         return cls
 
+    def get_view_names(self, connection: ConnectionWrapper, **kwargs: Any) -> List[str]:
+        s = "SELECT name FROM sqlite_master " "WHERE type='view' ORDER BY name"
```

Comment:
nitpick: the two strings exists in the original code, but serves no purpose - just join them?